### PR TITLE
Network setup

### PR DESCRIPTION
**Pull request by Kamill90**
Simulator Screen Shot - iPhone 14 Pro - 2022-11-07 at 15 44 56

### CODE REVIEW

File: src/screens/HomeScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      cityName={item.name}
+      description={item.description}
+      temperature={item.temp}
+      onPress={() => Alert.alert(item.name)}
+    />
+  );
+
+  return (
+    <View style={styles.container}>
+      {weatherData && (
```
Comment (adamTrz) [inline src/screens/HomeScreen.tsx:50]: What if there's no data present? 
Maybe we could show some loading indicator for the users?

File: .prettierrc.js [hunk 2]
```diff
@@ -1,7 +1 @@
-module.exports = {
-  arrowParens: 'avoid',
-  bracketSameLine: true,
-  bracketSpacing: false,
-  singleQuote: true,
-  trailingComma: 'all',
-};
+module.exports = {};
```
Comment (adamTrz) [inline .prettierrc.js:1]: Why this change? 🤔

Comment (Kamill90) [reply .prettierrc.js:1]: causing conflicts with eslint. Trailing comas are still on. No need.

File: src/screens/HomeScreen.tsx (hunk 1 above)
Comment (Kamill90) [reply src/screens/HomeScreen.tsx]: done

File: src/screens/HomeScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+      const normalizeWeatherData: any = [];
+      res.data.list.forEach((item) => {
+        normalizeWeatherData.push({
+          name: item.name,
+          dataId: item.id,
+          ...item.weather[0],
+          ...item.main,
+        });
+      });
```
[the remark this answers is not in the document]
Comment (Kamill90) [reply src/screens/HomeScreen.tsx]: done

**Review by adamTrz — APPROVED** [review]
